docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md

```diff
@@ -66,5 +66,90 @@ Alternatively, you may add the domain to the allowlist from the `config/Shared/c
+
+| schema.org property                        | Bazaarvoice property | Only for a Marketplace |
+|--------------------------------------------|----------------------|------------------------|
+| invoice.email                              | email                |                        |
+| invoice.priceCurrency                      | currency             |                        |
+| invoice.identifier                         | orderId              |                        |
+| invoice.total                              | price                |                        |
+| invoice.orderItem.price                    | items.price          |                        |
+| invoice.orderItem.orderQuantity            | items.quantity       |                        |
+| invoice.orderItem.sku                      | items.productId      |                        |
+| invoice.orderItem.name                     | items.name           |                        |
+| invoice.orderItem.offers.seller.name       | items.productId      | *                      |
+| invoice.orderItem.offers.seller.identifier | items.name           | *                      |
+
+#### Ratings and Reviews (Product)
+Core Templates:
+* `SprykerShop/Yves/ProductDetailPage/Theme/default/views/pdp/pdp.twig`
+* `SprykerShop/Yves/ProductReviewWidget/Theme/default/views/pdp-review-rating/pdp-review-rating.twig`
+* `SprykerShop/Yves/ProductReviewWidget/Theme/default/components/organisms/review-summary/review-summary.twig`
+
+| schema.org property         | Class name                |
+|-----------------------------|---------------------------|
+| product.sku                 |                           |
+| aggregateRating.ratingValue |                           |
+| aggregateRating.bestRating  |                           |
+|                             | review-summary            |
+
+#### Ratings and Reviews (Merchant)
+Core Template: `SprykerShop/Yves/MerchantProfileWidget/Theme/default/components/molecules/merchant-profile/merchant-profile.twig`
+
+| schema.org property         | Class name                |
+|-----------------------------|---------------------------|
+| aggregateRating             |                           |
+|                             | review-summary            |
+
+#### Inline Ratings
+Core Templates:
+* `SprykerShop/Yves/ShopUi/Theme/default/components/molecules/product-item/product-item.twig`
+* `SprykerShop/Yves/ProductReviewWidget/Theme/default/views/product-review-display/product-review-display.twig`
+* `SprykerShop/Yves/ProductReviewWidget/Theme/default/components/molecules/rating-selector/rating-selector.twig`
+
+| schema.org property         | Class name      |
+|-----------------------------|-----------------|
+| product.sku                 |                 |
+| aggregateRating.ratingValue |                 |
+| aggregateRating.bestRating  |                 |
+|                             | rating-selector |
```

Comment:
How should these tables be read? I don't understand the meaning of these tables.

Reply:
just a list of properties and classes that should be added, how would you suggest describing this?

Reply:
Summary: We will need to provide examples of what HTML markup must be present for customers who are making their own frontend (or significant changes to the default spryker frontend).